source/_components/insteon_local.markdown

```diff
@@ -44,14 +44,7 @@ insteon_local:
   password: YOUR HUB PASSWORD
   timeout: 10
   port: 25105
-
-light:
-  - platform: insteon_local
-  
-switch:
-  - platform: insteon_local
-  
-fan:
-  - platform: insteon_local
 ```
 
+In the most recent update we changed the naming of devices to match the insteon address. This means that the automations and scenes will have to be updated. You can also set friendly names for the devices using the customization option.  https://home-assistant.io/docs/configuration/customizing-devices/
```

Comment:
Same here. This doesn't need to be on the docs page, but this is a great write-up for the breaking changes section of the blog. If you can just include this in the PR description we'll copy it to the blog notes.